core/src/mindustry/content/Blocks.java

```diff
@@ -3043,6 +3058,9 @@ public static void load(){
             unitCapModifier = 15;
             researchCostMultipliers.put(Items.silicon, 0.4f);
             researchCostMultiplier = 0.1f;
+            
+            lightRadius = 150;
```

Comment:
These values shouldn't need to be manually defined in `Blocks`, keep the changes to CoreBlock only.